src/js/ui/menu.js

```diff
@@ -157,6 +157,7 @@ Menu.prototype._resolveMenu = function(clear, pushing) {
 Menu.prototype._resolveSection = function(e, clear) {
   var section = this._getSection(e);
   if (!section) { return; }
+  section = myutil.shadow({textColor: this.state.textColor, backgroundColor: this.state.backgroundColor}, section);
```

Comment:
The two properties can have their own line, I would keep it one line if there was just one property.

```
  section = myutil.shadow({
    textColor: this.state.textColor,
    backgroundColor: this.state.backgroundColor,
  }, section);
```